docs/LoaderInterfaceArchitecture.md

```diff
@@ -58,21 +58,21 @@ wanting a better understanding of the Vulkan runtime.
 
 The application sits at the top and interfaces directly with the Vulkan
 loader.
-At the bottom of the stack sits the implementations.
-An implementation can control one or more physical devices capable of rendering
-Vulkan, implement a conversion from Vulkan into a native graphics API (like
+At the bottom of the stack sits the drivers.
+A drivercan control one or more physical devices capable of rendering Vulkan,
```

Comment:
```suggestion
A driver can control one or more physical devices capable of rendering Vulkan,
```

Reply:
That was totally there to make sure you reviewed my code.  You passed!   ;-)